scripts/run_all_tests.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+shout() {
+  set +x
+  echo -e "\n!!!!!!!!!!!!!!!!!!!!\n${1}\n!!!!!!!!!!!!!!!!!!!!\n"
+  set -x
+}
+
+set -ex
+
+shout "Setting up"
+
+mkdir bin
+GOBIN="`pwd`/bin"
+KUBECONFIG="`pwd`/config"
+SCRIPT_IDENTITY=${SCRIPT_IDENTITY:-"def-id"}
+export SKIP_USER_LOGIN_TESTS="true"
+export GINKGO_TEST_ARGS="--noColor"
```

Comment:
Colour is cool, no? Makes it easy to read logs, no? Why no colour? :sob: 

Seriously, why no colour? :thinking:

Reply:
Because jenkins craps itself with colored output.

Reply:
Haha. Okay.